README.md

```diff
@@ -127,24 +127,35 @@ Returns the value of the first DOM node with `dataAttributeName`
 getData("some-attribute"); // "true"
 ```
 
-### isElementInViewport(_node_)
+### isEqual(_a, b_)
+
+_@creuna/utils/is-equal_
+
+* `a`: any
+* `b`: any
+* returns: boolean
+
+Checks whether `a` and `b` are equal (deep comparison for objects and arrays). This uses `JSON.stringify`, so be aware that array elements or object values that are `undefined` will be stripped.
+
+### isFullyInViewport(_node_)
 
-_@creuna/utils/is-element-in-viewport_
+_@creuna/utils/is-fully-in-viewport_
 
 * `node`: DOM node
 * returns: boolean
 
-Checks whether the given element is fully visible in the viewport
+Checks whether the given element is fully visible in the viewport. This is a special case of `isInViewport` where the offsets are the dimensions of the element.
 
-### isEqual(_a, b_)
+### isInViewport(_node, offset, offsetX_)
```

Comment:
`isInViewport(_node, offset, offsetX_)` should be `isInViewport(_node, offset[, offsetX]_)`, since `offsetX` is optional.